.travis.yml

```diff
@@ -8,8 +8,11 @@ node_js:
   - "iojs-v3"
   - "4"
   - "5"
+  - "6"
 env:
   - BROWSER=1
```

Comment:
Instead of having different `BROWSER_*` env vars, maybe it's simpler to just have a single `BROWSER`, and then have 3 different cases?
```
- BROWSER=bluebird
- BROWSER=local
- BROWSER=polyfill
```